src/main/res/layout/appwidget_dark.xml

```diff
@@ -57,15 +58,15 @@
     <ImageView
         android:layout_height="1sp"
         android:layout_width="fill_parent"
-        android:background="#0099cc"
+        android:background="@color/simple_primary"
         tools:ignore="ContentDescription,SmallSp"
         />
     <ListView
         android:id="@+id/widgetlv"
         android:layout_width="fill_parent"
         android:layout_height="fill_parent"
-        android:background="#000000"
-        android:divider="#444444"
+        android:background="@color/black"
+        android:divider="@color/gray81"
```

Comment:
The gray is a slightly different color but it doesn't really make a difference in how it looks.